<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSearchCriteria</class>
 <widget class="QDialog" name="DialogSearchCriteria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sucheinstellungen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-image: url(:/patterns/tex_grey);
background-repeat: repeat-xy;

}

QGroupBox {

 margin: 9px 9px 9px 9px;
 border: 1px solid rgb(144,255,23);
 border-radius: 2px; 
 margin-top: 9px;
 margin-bottom: 7px;
 padding: 0px;

background-image: url(:/patterns/woven);
background-repeat: repeat-xy;
} 

QGroupBox::title {
top:-9 ex;
left: 10px;
color: green;
subcontrol-origin: border;

padding: 2px;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;

background-image: 	url(:/patterns/carbon_dark);				
background-repeat: repeat-xy;
}

QLabel
{
	color: white;
}

QScrollArea
{
	background: none;
}

QWidget#headerGeneral
{
	background: url(:/patterns/carbon_dark);
}

QLabel#headerGeneralTitleLbl
{
background: qlineargradient(spread:pad, x1:0.178, y1:0, x2:1, y2:0, stop:0 rgba(37, 202, 0, 234), stop:1 rgba(255, 255, 255, 0));
padding-left: 9px;
color: black;
}

QWidget#headerLangPattern
{
	background: url(:/patterns/carbon_dark);
}

QLabel#headerLangPatternTitleLbl
{
background: qlineargradient(spread:pad, x1:0.178, y1:0, x2:1, y2:0, stop:0 rgba(37, 202, 0, 234), stop:1 rgba(255, 255, 255, 0));
padding-left: 9px;
color: black;
}

QWidget#headerCodeDescr
{
	background: url(:/patterns/carbon_dark);
}

QLabel#headerCodeDescrTitleLbl
{
background: qlineargradient(spread:pad, x1:0.178, y1:0, x2:1, y2:0, stop:0 rgba(37, 202, 0, 234), stop:1 rgba(255, 255, 255, 0));
padding-left: 9px;
color: black;
}

QLineEdit
{
background: rgb(204, 204, 204);
border: 1px solid black;
}

QScrollArea
{
background: rgb(204, 204, 204);
border: 1px solid black;
}

QDateEdit
{
background: rgb(204, 204, 204);
border: 1px solid black;
}

QTextEdit
{
background: rgb(204, 204, 204);
border: 1px solid black;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerGeneral" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="headerGeneralTitleLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Allgemeine Informationen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutNameDate">
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gBoxNameAuthor">
       <property name="title">
        <string>Name und Autor</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>14</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lblEntryName">
            <property name="text">
             <string>Name des Eintrags</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lEditEntryName">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>204</red>
                  <green>204</green>
                  <blue>204</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgb(204, 204, 204);
border: 1px solid black;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblNameAuthor">
            <property name="text">
             <string>Name des Autors</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lEditAuthorName"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gBoxDate">
       <property name="title">
        <string>Datum des Eintrags</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>14</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblFrom">
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>Von: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="datEditFrom">
            <property name="styleSheet">
             <string notr="true">background: transparent;
border: none;
color: white;
</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblTo">
            <property name="text">
             <string>Bis:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="datEditTo">
            <property name="styleSheet">
             <string notr="true">background: transparent;
border: none;
color: white;
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="headerLangPattern" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="headerLangPatternTitleLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sprache und Pattern</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutLangPattern">
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gBoxLanguage">
       <property name="title">
        <string>Sprache</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>14</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: none;
color: white;
</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaContentLanguage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>514</width>
             <height>69</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="vertLayoutContentLanguage"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gBoxPattern">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Pattern</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>14</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: none;
color: white;
</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaContentPattern">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>514</width>
             <height>69</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="vertLayoutContentPattern"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="headerCodeDescr" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="headerCodeDescrTitleLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Code &amp; Beschreibung</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutCodeDescr">
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gBoxCode">
       <property name="title">
        <string>Code</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="topMargin">
         <number>14</number>
        </property>
        <item>
         <widget class="QTextEdit" name="tEditCode">
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: none;
color: white;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gBoxDescr">
       <property name="title">
        <string>Beschreibung</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="topMargin">
         <number>14</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QTextEdit" name="tEditDescr">
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: none;
color: white;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bttnGo">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;
color: rgb(122, 182, 10);
}

QPushButton:pressed
{

background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(116, 202, 18);
border-style: outset;
color: green;
}</string>
       </property>
       <property name="text">
        <string>Los</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
